CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+### Mapbox Navigation SDK 1.0.0-rc.8 - September 9, 2020
+
+Visit our [1.0.0 migration guide](https://github.com/mapbox/mapbox-navigation-android/wiki/1.0-Navigation-SDK-Migration-Guide) and the [documentation pages](http://docs.mapbox.com/android/beta/navigation/overview) for more information.
+
+#### Changelog
+Changes since Mapbox Navigation SDK `1.0.0-rc.7`:
+* Add InstructionView attribute to allow developers to not show InstructionListView [#3537](https://github.com/mapbox/mapbox-navigation-android/pull/3537)
+* Use raw location samples in telemetry traces [#3533](https://github.com/mapbox/mapbox-navigation-android/pull/3533)
```

Comment:
This https://github.com/mapbox/mapbox-navigation-android/pull/3533 only landed in `1.1-alpha.2`. We should remove.